C-Plus-Plus/other/merge_2_sorted_arrays_without_extra_space.cpp

```diff
@@ -0,0 +1,60 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main() {
+
+  int m, n;
+  cout << "Enter the total number of elements in array1 and array2 respectively\n";
+  cin >> m >> n;
+
+  int arr1[m], arr2[n];
+  cout << "enter 1st array elements\n";
+  for (int i = 0; i < m; i++) {
+    cin >> arr1[i];
+  }
+  cout << "enter 2nd array elements\n";
+  for (int i = 0; i < n; i++) {
+
+    cin >> arr2[i];
+  }
+  
+  // iterating through first array from the end and the second array from start.
+  for (int i = m - 1, j = 0; i >= 0 && j < n; i--, j++) {
+    int temp;
+    // swap if number in 1st array is greater than 2nd array.
+    if (arr1[i] > arr2[j]) {
```

Comment:
@rutujadhanawade Since you have imported  `#include<bits/stdc++.h>`  so you can use `std::swap() ` function.

Reply:
added that function now.